msg/onboard_computer_status.msg

```diff
@@ -0,0 +1,23 @@
+# ONBOARD_COMPUTER_STATUS message data
+uint64 timestamp # time since system start (microseconds)
+uint32 uptime	 # time since system boot of the companion (milliseconds)
+
+uint8 type	 # type of onboard computer 0: Mission computer primary, 1: Mission computer backup 1, 2: Mission computer backup 2, 3: Compute node, 4-5: Compute spares, 6-9: Payload computers.
+
+uint8[8] cpu_cores # CPU usage on the component in percent
+uint8[10] cpu_combined # Combined CPU usage as the last 10 slices of 100 MS
+uint8[4] gpu_cores # GPU usage on the component in percent
+uint8[10] gpu_combined # Combined GPU usage as the last 10 slices of 100 MS
+int8 temperature_board # Temperature of the board
+int8[8] temperature_core # Temperature of the CPU core
+int16[4] fan_speed # Fan speeds
+uint32 ram_usage # Amount of used RAM on the component system
```

Comment:
Can you add the units everywhere where it makes sense?

Reply:
Done!